drivers/tests/TESTS/mbed_drivers/buffered_serial/CMakeLists.txt

```diff
@@ -3,11 +3,14 @@
 
 cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
 
+set(MBED_TEST_LINK_LIBRARIES "mbed-os" CACHE STRING "")
```

Comment:
we do not need to set this, it should be set if not yet defined in the greentea cmake?